pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -148,38 +141,26 @@ func (self *SuggestionsHelper) GetFilePathSuggestionsFunc() func(string) []*type
 		// doing another fuzzy search for good measure
 		matchingNames = utils.FuzzySearch(input, matchingNames)
 
-		suggestions := make([]*types.Suggestion, len(matchingNames))
-		for i, name := range matchingNames {
-			suggestions[i] = &types.Suggestion{
-				Value: name,
-				Label: name,
-			}
-		}
-
-		return suggestions
+		return matchesToSuggestions(matchingNames)
 	}
 }
 
 func (self *SuggestionsHelper) getRemoteBranchNames(separator string) []string {
-	result := []string{}
-	for _, remote := range self.model.Remotes {
-		for _, branch := range remote.Branches {
-			result = append(result, fmt.Sprintf("%s%s%s", remote.Name, separator, branch.Name))
-		}
-	}
-	return result
+	return slices.FlatMap(self.model.Remotes, func(remote *models.Remote) []string {
+		return slices.Map(remote.Branches, func(branch *models.RemoteBranch) string {
+			return fmt.Sprintf("%s%s%s", remote.Name, separator, branch.Name)
+		})
+	})
```

Comment:
I can see an argument in favour of the original code here, but we _are_ doing a flat map whether we specify it explicitly or not